NitroxClient/MonoBehaviours/Gui/MainMenu/LoadingScreenExtras.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using UnityEngine;
+
+namespace NitroxClient.MonoBehaviours.Gui.MainMenu
+{
+    public class LoadingScreenExtras
+    {
+        private static string assemblyVersion => Assembly.GetAssembly(typeof(LoadingScreenExtras)).GetName().Version.ToString();
+        private static GameObject loadingTextGameObject => uGUI.main.loading.loadingText.gameObject;
+
+        private static Dictionary<string, uGUI_TextFade> nitroxLoadingExtrasGameObject = new Dictionary<string, uGUI_TextFade>();
+
+        public static void Initialize()
```

Comment:
Do items in this class need to be static?  Can we model this as a monobehaviour?

Reply:
Because monobehaviour is not needed I think it would be better for performance to not make this a monobehaviour .